src/pmdas/amdgpu/localdrm.c

```diff
@@ -0,0 +1,323 @@
+/*
```

Comment:
This file could be renamed drm.c (and likewise the accompanying .h) - we don't need to perform any shared library dlopen/shim gymnastics here like we do with nvidia.